src/oled_display_node.cpp

```diff
@@ -519,20 +531,21 @@ int dispOled_setCursor(dispCtx_t *dispCtx, int column, int line) {
 int dispOled_clearDisplay(dispCtx_t *dispCtx) {
         int retCode = 0;
 
-        uint8_t zero[128];
+        uint8_t zero[140];
         zero[0] = OLED_CONTROL_BYTE_DATA_STREAM;;
-        for (uint8_t idx = 1; idx < 128; idx++) {
+        for (uint8_t idx = 1; idx < 136; idx++) {
```

Comment:
Based on old code, shouldn't this be `idx < 140`?

Reply:
No not a goof. It is in fact part of the fix.  Needed to clear 4 more bytes.  The buffer is len 140 so no harm done.
Often I use constants but this is so local to just this spot that I hard coded them.